pdf_statement_reader/parse.py

```diff
@@ -86,28 +91,33 @@ def clean_trans_detail(df, config):
             df.loc[i - 1, trans_detail] = row[trans_type]
 
 
-def clean_unwrap(df, config):
-    key = config["columns"][config["cleaning"]["unwrap"][0]]
-    value = config["columns"][config["cleaning"]["unwrap"][1]]
-    j = 0
-    for i, row in df.iterrows():
-        if pd.isnull(row[key]):
-            df.loc[j, value] += " " + df.loc[i, value]
-        else:
-            j = i
-
-
 def clean_date(df, config):
     date_cols = [config["columns"][col] for col in config["cleaning"]["date"]]
     if "date_format" in config["cleaning"]:
         date_format = config["cleaning"]["date_format"]
     else:
         date_format = None
 
-    year = df.iloc[0, 1][0:4]
+    cba = False  # json setting needed
```

Comment:
Need design decision how to handle unusual functionality

Reply:
Done in code https://github.com/marlanperumal/pdf_statement_reader/pull/50/commits/44c948f1e8b371c58e186ce5edf325958bbd7c34